components/link/link.js

```diff
@@ -113,10 +114,7 @@ class Link extends LocalizeCoreElement(FocusMixin(LitElement)) {
 					display: inherit;
 				}
 				a.truncate {
-					-webkit-box-orient: vertical;
-					display: -webkit-box;
-					overflow: hidden;
-					overflow-wrap: anywhere;
+					${getOverflowDeclarations({ lines: 1 })}
```

Comment:
The function requires a truthy `lines` value, so we give it `1`, but the actual line clamp style is updated inline in `render` to match `this.lines`.